src/lib.rs

```diff
@@ -597,6 +597,190 @@ impl<A: Array> ArrayVec<A> {
     }
 }
 
+impl<A: Array> ArrayVec<A>
+    where A::Item: Clone,
+{
+    /// Resizes the `ArrayVec` in-place so that `len` is equal to `new_len`.
+    ///
+    /// Does the same thing as `try_resize`, but panics on error instead of
+    /// returning it as a `Result`.
+    pub fn resize(&mut self, new_len: usize, value: A::Item) {
+        self.try_resize(new_len, value).unwrap()
+    }
+
+    /// Resizes the `ArrayVec` in-place so that `len` is equal to `new_len`.
```

Comment:
Rust's own formulation in their docs "so that `len` is equal" is awkward. Sounds better to just say "so that its length becomes `new_len`" or similar.